### PR TITLE
Add TorGuard

### PR DESCRIPTION
**Pull request by jonasrosland**
Adding TorGuard VPN client

### CODE REVIEW

File: Casks/torguard.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'torguard' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://updates.torguard.biz/Software/MacOSX/TorGuard-latest.dmg'
```
Comment (adidalal) [inline Casks/torguard.rb]: Needs a comment `# torguard.biz was verified as official when first introduced to the cask

**Comment by adidalal** [comment]
This can be a versioned URL - `"http://updates.torguard.biz/Software/MacOSX/TorGuard-v#{version}.dmg"`, with a version of `0.3.42`

Use http://updates.torguard.biz/Software/MacOSX/checksums.sha256 as the appcast.

**Comment by jonasrosland** [comment]
Hi @adidalal, I've added versioning and the app cast stanza, thanks for the heads up! Does this look better now?

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 8112452e85b47fa840468da6e816e90c6281c1da. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by jonasrosland** [comment]
Perfect, thank you!

**Pull request closed without merging**